src/VisualStudio/CSharp/Impl/LanguageService/CSharpHelpContextService.cs

```diff
@@ -284,7 +284,8 @@ private static bool TryGetTextForContextualKeyword(SyntaxToken token, out string
                             text = "partialmethod_CSharpKeyword";
                             return true;
                         }
-                        else if (token.Parent.GetAncestorOrThis<ClassDeclarationSyntax>() != null)
+                        else if (token.Parent.GetAncestorOrThis<ClassDeclarationSyntax>() != null ||
+                            token.Parent.GetAncestorOrThis<RecordDeclarationSyntax>() != null)
```

Comment:
@davidwengier Can we remove this else if altogether and assume "partialtype_CSharpKeyword" if SyntaxKind.PartialKeyword is found on non-`MethodDeclarationSyntax`.

Reply:
what about structs/interfaces?

Reply:
@CyrusNajmabadi I should have caught this from my docs PR 😄 
I would prefer removing the else if part then, what do you think?

Reply:
Seems like TypeDeclarationSyntax covers all bases